tests/postman/package.json

```diff
@@ -22,6 +22,7 @@
     "commander": "^7.2.0",
     "is-url-superb": "^6.1.0",
     "ky-universal": "^0.9.1",
+    "md5": "^2.3.0",
```

Comment:
sadge

Reply:
sha would be better

Reply:
Updated to replace MD5 with SHA256 from crypto-js module